Added the functions except log out

PatsiChang opened the pull request:
The log out function need to wait for backend implementation (RECIPE-93)
Link: https://smart-home-cloud.atlassian.net/browse/RECIPE-90

PatsiChang committed:
Added the functions except log out

PatsiChang committed:
Added logout Functions

ChakShingLo commented on new/app/loginPage/index.tsx:
```diff
@@ -33,8 +32,8 @@ export default function LoginPage() {
-                <BasicTextInput name="uid" label="User Id" type="username"/>
-                <BasicTextInput name="password" label="Password" type="password"/>
+                <BasicTextInput name="userId" label="User Id" type="username" />
+                <BasicTextInput name="logInPasswordHashed" label="Password" type="password" />
```
why it is named logInPasswordHashed? think the value is not hashed yet?

PatsiChang replied:
This is fixed

ChakShingLo commented on new/lib/userSessionApi.tsx:
```diff
@@ -15,23 +18,27 @@ function recoverToken() {
+    // await new Promise((r) => setTimeout(r, 1000));
+    const person: UserLogin = {
+        userId: uid,
+        logInPasswordHashed: password,
```
will the password be hashed on client-side or server-side?

PatsiChang replied:
you are right, at this moment its not hashed, but i think we will hash it before we sent it to the server by encryption?

PatsiChang committed:
Fix naming

PatsiChang committed:
Fix naming

Accidentally changed this password too in the past commit, this one should remain logInPasswordHashed as the formData mocked logInPasswordHashed
